### ISSUE TITLE
No media_group_id in response

### ISSUE THREAD

**Issue by midnightcoder-pro** [tdlib/telegram-bot-api#46]
Hi, I have a problem with quickly dropping mediaGroup into a channel. Bot in linked chat gets media_group_id every other time. Will there be a fix for this?

**Comment by levlam** [comment]
Could you provide an example of the issue, including updates received by the bot?

**Comment by midnightcoder-pro** [comment]
`https://api.telegram.org/botToken/getUpdates?allowed_updates=photo` returns normally result, but sometimes there is no media_group_id field. Or should I create blank bot, channel and linked group to provide an example?

<details>
<summary>getUpdates (personal information removed)</summary>

```json5
[
	{
		"channel_post": {
			"message_id": 735,
			"sender_chat": {
				"type": "channel"
			},
			"chat": {
				"type": "channel"
			},
			"media_group_id": "replaced", // here it is 
			"photo": []
		}
	},
	{
		"channel_post": {
			"message_id": 736,
			"sender_chat": {
				"type": "channel"
			},
			"chat": {
				"type": "channel"
			},
			"media_group_id": "replaced", // here it is too
			"photo": []
		}
	},
	{
		"message": {
			// there is no media_group_id field here 😥
			"from": {
				"id": 777000,
				"is_bot": false,
				"first_name": "Telegram"
			},
			"sender_chat": {
				"type": "channel"
			},
			"chat": {
				"type": "supergroup"
			},
			"forward_from_chat": {
				"type": "channel"
			},
			"forward_from_message_id": 735,
			"photo": []
		}
	},
	{
		"message": {
			// here too 😥
			"from": {
				"id": 777000,
				"is_bot": false,
				"first_name": "Telegram"
			},
			"sender_chat": {
				"type": "channel"
			},
			"chat": {
				"type": "supergroup"
			},
			"forward_from_chat": {
				"type": "channel"
			},
			"forward_from_message_id": 736,
			"photo": []
		}
	}
]
```
</details>

It seems to me that if you do not go to the comments immediately, the field will be present

**Comment by luckydonald** [comment]
Would `allowed_updates=media_group_id` work?

**Comment by midnightcoder-pro** [comment]
There is no `media_group_id` update type in https://core.telegram.org/bots/api#update

**Comment by luckydonald** [comment]
But there's no `photo` either...

**Comment by midnightcoder-pro** [comment]
It doesn't matter, it still gets data with an error
(I use telegraf nodejs lib, that's why I added their subtype to the query out of habit)

**Comment by levlam** [comment]
@midnightcoder-pro Could you send full received updates with all fields to @tdlib_bot in Telegram?

**Comment by midnightcoder-pro** [comment]
@levlam ok, I sent

**Comment by levlam** [comment]
@midnightcoder-pro Thanks! Fixed in Bot API 5.0.1.
